lom/int_test/config.txt

```diff
@@ -0,0 +1,4 @@
+remote_host=10.3.147.17
```

Comment:
You are running within the container.
You don't need these.
The client APIs use localhost to connect

Reply:
As agreed , will revisit integration tests later